charts/rancher-monitoring/v0.0.7000/charts/webhook-receiver/templates/service.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: webhook-receiver
+  labels:
+    app: {{ template "app.name" . }}
+    chart: {{ template "app.version" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+spec:
+  type: ClusterIP
+  selector:
+    app: {{ template "app.name" . }}
+    chart: {{ template "app.version" . }}
```

Comment:
remove the `chart: {{ template "app.version" . }}` selector if the above changes are made.